plans/terraform/apply.pp

```diff
@@ -77,6 +76,11 @@
   Optional[Enum['linux', 'windows']] $os_type,
   Optional[Provision::ProviderOptions] $provider_options,
 ) {
+  # General prechecks
+  if $pe_server == undef {
+    out::message('pe_server value is undef, will not able to configure the provisioned node.')
```

Comment:
Maybe better to use log::warn here? - https://www.puppet.com/docs/bolt/latest/plan_functions.html#logwarn